### DISCUSSION TITLE
possible values for the "cold" column

### DISCUSSION THREAD

**Discussion by tabascoeye** [teslamate-org/teslamate#694] [Ideas]
**What would you like to be added**:
Documentation for the "cold" column which is shown in the "drives" table.

**Why is this needed**:
To understand, what is meant here

**Additional context**:
screenshot:
<img width="295" alt="Screenshot 2020-05-06 at 14 13 13" src="https://user-images.githubusercontent.com/111893/81175559-06d12600-8fa4-11ea-8b89-5291741c80f1.png">

Difference between "empty" and "-" is unclear

(feeld free to insta-close this issue if I'm just too dumb to find the explanation)

**Comment by Dulanic** [comment]
It is basically when the usable range is lower than the expected range. This commonly happens when it's cold outside. The difference between - and blank is null vs false... the null ones happen because the usable battery level was null which can happen for various reasons. I'll put in a PR so null and false show as the same.
